src/main/scala/nodes/learning/BlockLinearMapper.scala

```diff
@@ -18,24 +18,24 @@ import workflow.{Transformer, LabelEstimator}
  * @param bOpt optional intercept term to be added
  * @param featureScalersOpt optional seq of transformers to be applied before transformation
  */
-class BlockLinearMapper(
+class BlockLinearMapper[T <: Vector[Double]](
     val xs: Seq[DenseMatrix[Double]],
     val blockSize: Int,
     val bOpt: Option[DenseVector[Double]] = None,
     val featureScalersOpt: Option[Seq[Transformer[DenseVector[Double], DenseVector[Double]]]] = None)
```

Comment:
Does it make sense for this to take `Option[Seq[Transformer[DenseVector[Double], DenseVector[Double]]]]]` or `O[S[T[Vector[Double], Vector[Double]]]]`